public/main.js

```diff
@@ -108,8 +129,46 @@ findPetButton.addEventListener('click', (event) => {
   }
 })
 
+postButton.addEventListener('click', (event) => {
+  var petToPost = {
+    type: addType.value,
+    name: addName.value,
+    breed: addBreed.value,
+    age: addAge.value,
+    gender: addGender.value,
+    image: addImage.value
+  }
+
+  if (noCatsCheckbox.checked === true) {
+    petToPost.noCats = true
+  }
```

Comment:
Small suggestion here, `petToPost.noCats` needs to hold the value of `noCatsCheckbox.checked`, so we can simply do:

```javascript
petToPost.noCats = noCatsCheckbox.checked
petToPost.noDogs = noDogsCheckbox.checked
```